backend/app/services/share/base_service.py

```
@@ -323,9 +373,30 @@ def create_share_link(
         db.add(share_link)
-        db.commit()
+
+        try:
+            db.commit()
+            logger.info(
+                f"[create_share_link] Successfully committed new share_link: id={share_link.id}"
+            )
+        except Exception as e:
+            logger.error(
+                f"[create_share_link] FAILED to commit share_link: {type(e).__name__}: {str(e)}"
+            )
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)